### DISCUSSION TITLE
Airtable to Oracle replication , how to handle multiselect field from Airtable to Oracle

### DISCUSSION THREAD

**Discussion by shahmheraein** [airbytehq/airbyte#32315] [Connector Questions]
Hi everyone,

I have setup replication between Airtable and Oracle and can sync the data to Oracle. One issue that I am running into is the multiselect field/column in Airtable table is not getting copied/synced to Oracle table. I am not sure if there is way to overcome this The Column type for the multiselect field in Airtable in target table in Oracle is to varchar2 by default and I am assuming that is why the data is not been copied over. 

Thanks

**Comment by marcosmarxm** [comment]
Do you have another source to Oracle to test the column selection and confirm if its a problem with Oracle or Airtable connector?

[a comment by an automated account is not shown]
